NEWS.md

```diff
@@ -1,5 +1,6 @@
 # riskassessment (development version)
 
+* Add pdf option to databaseView and reportPreview
```

Comment:
Maybe be a little more description would help here. Consider amending so it makes more sense for the reader since users won't know those shiny module names. Also, if you could please link to the github issue, that'd be helpful. For example:
>Added new pdf report download option to 'Database' and 'Report Preview' tabs (#300).

Reply:
OK, incremented version number and added `{tinytex}` to suggests field, and updated NEWS.md